apps/web/components/ConnectWalletButton.tsx

```diff
@@ -14,12 +14,13 @@ import { Fragment } from "react";
 import { formatAddress } from "@/utils/formatAddress";
 import { Menu, Transition } from "@headlessui/react";
 import { ChevronUpIcon, PowerIcon } from "@heroicons/react/24/solid";
+import { getChainIdFromPath } from "@/utils/path";
 
 export const ConnectWallet = () => {
   const path = usePathname();
   const account = useAccount();
-  const urlChainId = Number(path.split("/")[2]);
-  const tokenUrlAddress = path.split("/")[3];
+  const urlChainId = +getChainIdFromPath();
```

Comment:
plus sign (+) looks unecessary, you already convert it to Number

Reply:
![image](https://github.com/1Hive/gardens-v2/assets/13890618/4b988317-9c67-4e10-be1a-99404bd62e2a)

Reply:
Change getChainIdFromPath return type to number

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/707108277c131da4cb3496f05337c09540390770

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/5496912036e05bbb3274af4d9c822c308e3b2946